src/libs/CardUtils.ts

```diff
@@ -56,4 +56,25 @@ function maskCard(lastFour = ''): string {
     return maskedString.replace(/(.{4})/g, '$1 ').trim();
 }
 
-export {getDomainCards, getCompanyCards, getMonthFromExpirationDateString, getYearFromExpirationDateString, maskCard};
+/**
+ * Formats an address object into an easily readable string
+ *
+ * @returns - formatted address
+ */
+function getFormattedAddress(privatePersonalDetails: OnyxTypes.PrivatePersonalDetails): string | null {
+    const {address} = privatePersonalDetails;
+    const [street1, street2] = (address?.street ?? '').split('\n');
+    const addressItems = [street1, street2, address?.city, address?.state, address?.zip, address?.country];
```

Comment:
If we don't have Address Line 2 populated, it just shows a blank space instead of filtering it out

<img width="309" alt="image" src="https://github.com/Expensify/App/assets/9059945/7f13b7f6-dcb7-4f42-bbb8-31c0a6fce98b">

Could we just re-use [getFormattedAddress](https://github.com/Expensify/App/blob/dee163261d69acc5e8c7dcc3961c9f389c5f8fd9/src/pages/settings/Profile/PersonalDetails/PersonalDetailsInitialPage.js#L78)? Seems like both methods are supposed to be doing the same thing, we could move it to addressUtils (or similar) and use one method for both pages

Reply:
I changed it a little bit and moved it to `PersonalDetailsUtils`, let me know what you think!